docs/content/how-to/kubevirt/create-kubevirt-cluster.md

```diff
@@ -32,11 +34,42 @@ mv bin/hypershift /output/hypershift'
 sudo mv $PWD/hypershift /usr/local/bin
 ```
 
+### Method 2 - Extract HyperShift CLI from the Operator Image
+
+!!! note
+
+    We are using Podman in the example, same applies to Docker.
+
+~~~sh
+podman cp $(podman create --name hypershift --rm --pull always quay.io/hypershift/hypershift-operator:latest):/usr/bin/hypershift /tmp/hypershift && podman rm -f hypershift
+
+sudo install -m 0755 -o root -g root /tmp/hypershift /usr/local/bin/hypershift
+~~~
+
+## Deploy the HyperShift Operator
+
 Use the hypershift cli tool to install the HyperShift operator into the
 management cluster.
 
+!!! note
+
+    If we don't define the HyperShift image we want to use, by default the CLI will deploy `latest`. Usually you want to deploy the image matching the release of the OpenShift cluster where HyperShift will run.
+
 ```shell
+# This install latest
 hypershift install
+# You may want to run this instead
+hypershift install --hypershift-image quay.io/hypershift/hypershift-operator:4.12
```

Comment:
i think this has the potential to lead to more confusion. if someone uses a hypershift operator pinned to a specific release (and not the latest one) then we have to start talking about release payload versioning and other stuff, which isn't well defined at the moment. i'd rather we stick to defaults for the documentation examples.

Reply:
Got it, will remove that from the doc.